src/Component.php

```diff
@@ -231,6 +239,14 @@ protected function getVariantDefaultVariables(string $modifier, array $defined_v
    */
   protected function extractParts(string $compound_name): array {
     $loader = $this->env->getLoader();
+    if (stripos($compound_name, '@') !== FALSE && stripos($compound_name, '.twig') === FALSE) {
+      $exploded_paths = explode('/', $compound_name);
+      $exploded_paths = array_filter($exploded_paths);
+      $last_part = $exploded_paths[count($exploded_paths) - 1];
+      $last_part = str_replace('@', '', $last_part);
+      $compound_name .= '/' . $last_part . '.twig';
+    }
```

Comment:
Why are you trying to resolve the namespaced path here? The loader is doing that already?

Reply:
Hi, the namespace is resolved when the 'render' tag includes the twig file path as well but doesn't resolve when twig path is missing.
Ex:  `{% render "@card/card.twig" %}` is working
        `{% render "@card" %}` is not working
As per Fractal standards, the latter is supposed to auto pick the "card.twig" template from the component by default. So trying to insist that behaviour by checking if .twig exists in the path and auto populate the default template if not provided as in [https://github.com/netzstrategen/twig_fractal/pull/30/files/1729f0588a258f12685f1de04e2dc72a272326b8#diff-79446737c5fc656d037e5ebaee2e336acb25c73dfff3d03856d42728276b9b32R242](https://github.com/netzstrategen/twig_fractal/pull/30/files/1729f0588a258f12685f1de04e2dc72a272326b8#diff-79446737c5fc656d037e5ebaee2e336acb25c73dfff3d03856d42728276b9b32R242)

Reply:
Ah okay.  In that case, it's fine; we can just simplify the code.

Reply:
thanks, this change is added now